subiquity/common/types.py

```diff
@@ -346,7 +346,7 @@ class WSLConfigurationBase:
 
 
 @attr.s(auto_attribs=True)
-class WSLConfigurationAdvanced:
+class WSLConfigurationAdvanced(WSLConfigurationBase):
```

Comment:
WSLConfigurationBase and WSLConfigurationAdvanced are 2 independents models and controllers.

Where in reconfigure mode, we show both base and advanced screen, each one showing its own model.

Do no heritate from it, WSLConfigurationAdvanced are additional properties that are not in WSLConfigurationBase, or said differently, being in setup or reconfigure mode, custom_path, custom_mount_opt, gen_host and gen_revolvconf are always set in WSLConfigurationBase.

Reply:
Both are fixed